view/frontend/layout/catalog_category_view.xml

```diff
@@ -8,7 +8,7 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceBlock name="category.product.addto">
-            <block class="Magento\Catalog\Block\Product\ProductList\Item\Block" after="-" ifconfig="extend_plans/product_protection/offer_display_settings/enable_pdp_offer" name="extend.catalog.product.view.product-protection-modal-offer" template="Extend_Integration::catalog/product/view/product-protection-modal-offer.phtml">
+            <block class="Magento\Catalog\Block\Product\ProductList\Item\Block" after="-" ifconfig="extend_plans/product_protection/enable" name="extend.catalog.product.view.product-protection-modal-offer" template="Extend_Integration::catalog/product/view/product-protection-modal-offer.phtml">
```

Comment:
set all `ifconfig`s back to the primary `product_protection` toggle. if it's `off`, we don't want to even call the template. if it's `on`, then call the template which will make the final determination as to whether the offer is rendered, based on the relevant offer display setting.